rsciio/edax/_api.py

```diff
@@ -123,7 +129,7 @@ def get_spc_dtype_list(load_all=False, endianess="<", version=0.61):
     """
     Get the data type list for an SPC spectrum.
     Further information about the file format is available `here
-    <https://github.com/hyperspy/hyperspy/files/29506/SPECTRUM-V70.pdf>`__.
+    <https://github.com/hyperspy/rosettasciio/raw/main/docs/file_specification/edax/SPECTRUM-V70.pdf>`__.
```

Comment:
```suggestion
    <https://github.com/hyperspy/rosettasciio/blob/57b114cc02a0cbe022f1c50c1d59779a44952685/docs/file_specification/edax/SPECTRUM-V70.pdf>`__.
```

Reply:
The `raw` link directly loads the file, the `blob` loads the github page of the file -- do you really prefer that one?

Reply:
The ["blob" link is permanent](https://docs.github.com/en/repositories/working-with-files/using-files/getting-permanent-links-to-files), and it should exist as long as the commit exists on github. The other link will break when its location changes in the repository, or the branch name changes, etc.